python-bindings/pythonbindings.cpp

```diff
@@ -64,7 +77,7 @@ PYBIND11_MODULE(blspy, m)
             [](const PrivateKey &k, const py::object &memo) {
                 return PrivateKey(k);
             })
-        .def("get_g1", [](const PrivateKey &k) { return k.GetG1Element(); })
+        .def("get_g1", [](const PrivateKey &k) { PythonGIL release_lock; return k.GetG1Element(); })
```

Comment:
Indentation

Reply:
I don't see the indentation problem. It matches the surrounded expressions and uses spaces.